app/views/applications/new.html.erb

```diff
@@ -8,7 +8,13 @@
     }
   ]
 } %>
-<h1>Create new application</h1>
-<p class="lead">Applications are created automatically when first deployed, but you can create them manually if you like.</p>
```

Comment:
Is there any CSS associated with this class that needs removing?

Reply:
Nope, possibly a bootstrap thing? There are some later commits which remove bootstrap and any unused CSS.